src/components/UserProfile.jsx

```diff
@@ -128,6 +132,22 @@ export default function UserProfile({
           /> */}
         </CardContainer>
         <CardContainer>
+          <SightingsCard
+            title={
+              someoneElse ? (
+                <FormattedMessage
+                  id="USERS_UNPROCESSED_AGS"
+                  values={{ name }}
+                />
+              ) : (
+                <FormattedMessage id="PENDING_SIGHTINGS" />
+              )
+            }
```

Comment:
there is also a titleId prop. it's probably simpler to use that, though we might need to add support for values